crates/bevy_ui/src/render/render_pass.rs

```diff
@@ -215,8 +215,14 @@ impl<P: PhaseItem> RenderCommand<P> for DrawUiNode {
             return RenderCommandResult::Failure;
         };
 
-        pass.set_vertex_buffer(0, ui_meta.into_inner().vertices.buffer().unwrap().slice(..));
-        pass.draw(batch.range.clone(), 0..1);
+        let ui_meta = ui_meta.into_inner();
```

Comment:
Comments on this chunk of code would be helpful to explain what's going on.

Reply:
Would something like this be sufficient?

```rust
        // Store the vertices
        pass.set_vertex_buffer(0, ui_meta.vertices.buffer().unwrap().slice(..));
        // Define how to "connect" the vertices
        pass.set_index_buffer(
            ui_meta.indices.buffer().unwrap().slice(..),
            0,
            bevy_render::render_resource::IndexFormat::Uint32,
        );
        // Draw the vertices
        pass.draw_indexed(batch.range.clone(), 0, 0..1);
```

Or did you want something more comprehensive?

Reply:
Yep that's great :)

Reply:
Great, these are in already since I accidentally committed them :)